static/app/views/feedback/feedbackListPage.tsx

```diff
@@ -88,7 +88,7 @@ export default function FeedbackListPage({}: Props) {
                     <Container style={{gridArea: 'list'}}>
                       <FeedbackList />
                     </Container>
-                    <FeedbackSearch style={{gridArea: 'search'}} />
```

Comment:
why is this gone?

Reply:
We don't use the style prop anymore for the new search bar